docs/articles/documentation/test-api/actions/README.md

```diff
@@ -123,6 +123,11 @@ Before executing an action, TestCafe waits for the target element to appear
 in the DOM and become visible. If this does not happen
 within the [selector timeout](../selecting-page-elements/selectors/using-selectors.md#selector-timeout), the test fails.
 
+Note that TestCafe cannot interact with page elements overlayed by a different element.
```

Comment:
Maybe we should explain it using the `z-index` CSS property: [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/z-index).